conf/app.conf

```diff
@@ -3,7 +3,7 @@ httpport = 8000
 runmode = dev
 copyrequestbody = true
 driverName = mysql
-dataSourceName = root:root@tcp(localhost:3306)/
+dataSourceName = root:123456@tcp(localhost:3306)/
```

Comment:
why?

Reply:
This password is set the same as in docker-compose and github-workflows files